--- setup.py.orig	2020-02-13 19:14:40.000000000 +0000
+++ setup.py	2020-02-13 19:14:54.000000000 +0000
@@ -73,7 +73,7 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['numpy', 'scipy', 'matplotlib', 'h5py', 'h5py_cache'],       
+    install_requires=['numpy', 'scipy', 'matplotlib', 'h5py'],       
 
     # List additional groups of dependencies here (e.g. development
     # dependencies). You can install these using the following syntax,
--- ncempy/io/emdVelox.py.orig	2020-02-13 19:17:37.000000000 +0000
+++ ncempy/io/emdVelox.py	2020-02-13 19:20:58.000000000 +0000
@@ -12,8 +12,6 @@
 import numpy as np
 import h5py
 
-import h5py_cache
-
 class fileEMDVelox:
     '''Class to represent Velox EMD files. It uses the h5py_cache module
     to increase the default cache size from 1MB to 10MB. This significantly
@@ -70,8 +68,7 @@
 
         # try opening the file
         try:
-            #self.file_hdl = h5py.File(filename, 'r')
-            self.file_hdl = h5py_cache.File(filename, 'r', chunk_cache_mem_size=10*1024**2)
+            self.file_hdl = h5py.File(filename, 'r', rdcc_nbytes=10*1024**2)
         except:
             print('Error opening file for readonly: "{}"'.format(filename))
             raise
@@ -185,4 +182,4 @@
         md['detectorName'] = detectorName
         
         return md
-    
\ No newline at end of file
+    
